pkg/ansible/watches/watches.go

```diff
@@ -112,6 +125,7 @@ func (w *Watch) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	w.Playbook = tmp.Playbook
 	w.Role = tmp.Role
 	w.MaxRunnerArtifacts = tmp.MaxRunnerArtifacts
+	w.MaxWorkers = getMaxWorkers(gvk, maxWorkersDefault)
```

Comment:
This is where things get weird for me. Because we are unmarshaling yaml that is missing data. Then we are calling a function that ultimately goes out the environment to get the value. And the watch doesn't even care about this value.

Reply:
I'm not certain I follow what it means for the watch to care about the value. `maxWorkers` only makes sense as a controller option per watch, `role` and `playbook` only make sense when a runner is created from the watch. The watch itself doesn't specifically care about this value, but again you can override the `maxWorkers` for a particular GVK which is 1=1 with a watch...it seems reasonable to me for the value of `maxWorkers` to be on the watch struct. You load all of the watches and then the watches are used to create the runner, controller, start the manager, etc.